deps.windows/60-carla.ps1

```diff
@@ -0,0 +1,64 @@
+param(
+    [string] $Name = 'carla',
+    [string] $Version = '2.6.0-alpha1',
+    [string] $Uri = 'https://github.com/falkTX/Carla.git',
+    [string] $Hash = '31f9a6eef93f205a97deafd251dc4dcb6f5bf5c9'
+)
+
+function Setup {
+    Setup-Dependency -Uri $Uri -Hash $Hash -DestinationPath $Path
+}
+
+function Clean {
+    Set-Location $Path
+
+    if ( Test-Path "build_${Target}" ) {
+        Log-Information "Clean build directory (${Target})"
+        Remove-Item -Path "build_${Target}" -Recurse -Force
+    }
+}
+
+function Configure {
+    Log-Information "Configure (${Target})"
+    Set-Location $Path
+
+    $Options = @(
+        $CmakeOptions
+        '-DCARLA_USE_JACK=OFF'
```

Comment:
See above regarding cache variable type declaration.

Reply:
done for both. rebased against latest git master too